tests/playbooks/tests_ipv6_dns_search.yml

```diff
@@ -138,11 +138,11 @@
           assert:
             that:
               - __network_connections_result.stderr is search("Setting
-                'dns_search' or 'dns_options' is not allowed when both IPv4 and
-                IPv6 are disabled.")
+                'dns_search' or 'dns_options' or 'dns_priority' is not allowed
+                when both IPv4 and IPv6 are disabled.")
```

Comment:
nit: maybe it should be `dns_search, dns_options and dns_priority are not allowed..`

Reply:
I think that 'or' is better than 'and',  because the user may not set the `dns_search, dns_options and dns_priority` all at once, they may set one of them. What do you think ?

Reply:
Hm. That is right.. could we get a second opinion maybe? @thom311

Reply:
I would say `dns_search, dns_options and dns_priority are not allowed`.

But not strong opinion. Both seems fine.

Reply:
I updated.